examples/keras_recipes/reproducibility_recipes.py

```diff
@@ -0,0 +1,274 @@
+"""
+Title: Reproducibility in Keras Models
+Author: [Frightera](https://github.com/Frightera)
+Date created: 2023/05/05
+Last modified: 2023/05/05
+Description: Demonstration of random weight initialization and reproducibility in Keras models.
+Accelerator: GPU
+"""
+"""
+## Introduction
+This example demonstrates how to control randomness in Keras models. Sometimes
+you may want to reproduce the exact same results across runs, for experimentation
+purposes or to debug a problem.
+
+This tutorial applies to Keras 2.7 and higher.
+"""
+
+"""
+## Setup
+"""
+import inspect
+import json
+
+import tensorflow as tf
+from tensorflow import keras
+from tensorflow.keras import layers
+
+# Set the seed using keras.utils.set_random_seed. This will set:
+# 1) `numpy` seed
+# 2) `tensorflow` random seed
+# 3) `python` random seed
+keras.utils.set_random_seed(812)
+
+# This will make TensorFlow ops as deterministic as possible.
```

Comment:
perhaps mention the performance loss that follows from this to explain why its not on by default